src/Code.gs

```diff
@@ -646,12 +549,117 @@ const ChatGPTApp = (function () {
     }
 
     if (verbose) {
-      console.log('Got response from openAI API');
+      console.log(`Got response from openAI API ${numberOfAPICalls}`);
     }
 
     return responseMessage;
   }
 
+  function handleToolCalls(responseMessage, tools, messages) {
```

Comment:
Add js doc for this function, to correctly type each parameter + type of what is returned
(even if it's an internal function, not user facing)